Update connect frontend/supply chain pages and remove hello world page in SDK docs - closes #548 (#549)
LiskHQ/lisk-docs: modules/ROOT/pages/guides/broadcast.adoc

:url_lisk_wallet: https://lisk.io/wallet

:url_api: reference/api.adoc

> :url_guides_hello: guides/index.adoc#_the_hello_world_app

Reviewer: Please create a custom ID and use it here

Author: OK done..

Custom transaction::
+
--

> The `HelloTransaction` from the xref:{url_guides_hello}[Hello World App] is used as an example here.

Reviewer: ```suggestion
The `HelloTransaction` from the xref:{url_guides_hello}[Hello World app] is used as an example here.
```

> The `HelloTransaction` from the xref:{url_tutorials_hello}[Hello World guide] is used as an example here.

Reviewer: ```suggestion
The `HelloTransaction` of the xref:{url_tutorials_hello}[Hello World app] is used as an example here.
```

Custom transaction::
+
--

> The `HelloTransaction` from the xref:{url_guides_hello}[Hello World App] is used as an example here.

Reviewer: ```suggestion
The `HelloTransaction` from the xref:{url_guides_hello}[Hello World app] is used as an example here.
```

> The `HelloTransaction` from the xref:{url_tutorials_hello}[Hello World guide] is used as an example here.

Reviewer: ```suggestion
The `HelloTransaction` of the xref:{url_tutorials_hello}[Hello World app] is used as an example here.
```

Custom Transaction::
+
--

> The `HelloTransaction` from the xref:{url_guides_hello}[Hello World App] is used as an example here.

Reviewer: ```suggestion
The `HelloTransaction` from the xref:{url_guides_hello}[Hello World app] is used as an example here.
```

> The `HelloTransaction` from the xref:{url_tutorials_hello}[Hello World guide] is used as an example here.

Reviewer: ```suggestion
The `HelloTransaction` of the xref:{url_tutorials_hello}[Hello World app] is used as an example here.
```

Author: done...suggestion committed.
